test/export/CodeSystemExporter.test.ts

```diff
@@ -0,0 +1,102 @@
+import { CodeSystemExporter } from '../../src/export';
+import { FSHDocument, FSHTank } from '../../src/import';
+import { FshCodeSystem } from '../../src/fshtypes';
+import { FshConcept } from '../../src/fshtypes/FshConcept';
+
+describe('CodeSystemExporter', () => {
+  let doc: FSHDocument;
+  let input: FSHTank;
+  let exporter: CodeSystemExporter;
+
+  beforeEach(() => {
+    doc = new FSHDocument('fileName');
+    input = new FSHTank([doc], { name: 'test', version: '0.0.1', canonical: 'http://example.com' });
+    exporter = new CodeSystemExporter(input);
+  });
+
+  it('should output empty results with empty input', () => {
+    const exported = exporter.export();
+    expect(exported).toEqual([]);
+  });
+
+  it('should export a single code system', () => {
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    doc.codeSystems.set(codeSystem.name, codeSystem);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0]).toEqual({
+      name: 'MyCodeSystem',
+      id: 'MyCodeSystem',
+      status: 'active',
+      content: 'complete',
+      url: 'http://example.com/CodeSystem/MyCodeSystem'
+    });
+  });
+
+  it('should export a code system with additional metadata', () => {
+    const codeSystem = new FshCodeSystem('MyCodeSystem');
+    codeSystem.title = 'My Fancy Code System';
```

Comment:
Can you also set `id` to something different here?  I'm sure it works, but it's a common use case so let's reflect it in the tests.